src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js

```diff
@@ -351,6 +366,9 @@ SimplePowerBIReportEmbed.defaultProps = {
     hasErrors: {
       label: 'An error occured during the scenario run',
     },
+    hasUnknownStatus: {
+      label: 'This scenario has the "Unknown" status, if the problem persists, please, contact your administrator',
```

Comment:
End users don't have to be aware of our technical details (i.e. the "Unknown" status), we could write something like: "This scenario is in an unknown state, if the problem persists..."